tests/locator_test.go

```diff
@@ -410,3 +410,24 @@ func TestLocatorTap(t *testing.T) {
 		}, "should not select multiple elements")
 	})
 }
+
+//nolint:tparallel
+func TestLocatorDispatchEvent(t *testing.T) {
+	t.Parallel()
+
+	tb := newTestBrowser(t, withFileServer())
+	p := tb.NewPage(nil)
+	require.NotNil(t, p.Goto(tb.staticURL("/locators.html"), nil))
+
+	t.Run("ok", func(t *testing.T) {
+		result := func() bool {
+			ok := p.Evaluate(tb.toGojaValue(`() => window.result`))
+			return ok.(goja.Value).ToBoolean() //nolint:forcetypeassert
+		}
+		p.Locator("#link", nil).DispatchEvent("click", tb.toGojaValue("mouseevent"), nil)
+		require.True(t, result(), "could not dispatch event")
```

Comment:
Can you assert on the value of `result()` before dispatching the event?

Reply:
Makes sense.